jstest/flasher/flash.config

```diff
@@ -0,0 +1,60 @@
+{
+  "stm32f4dis": {
+    "flash": {
+      "cwd": "%{stlink}",
+      "cmd": "build/Release/st-flash",
+      "args": ["write", "%{build-dir}/nuttx.bin", "0x8000000"]
+    }
+  },
+  "artik053": {
+    "flash": {
+      "cwd": "%{tizenrt}/os",
+      "cmd": "make",
+      "args": ["download", "ALL"]
+    }
+  },
+  "artik530": {
+    "init": [
+      {
+        "condition": "'%{appname}' == 'iotjs'",
+        "cmd": "function(init_freya_config)",
```

Comment:
what if I use the no-memstat option? I think the init_freya_config does not need in those cases.

Reply:
You are right.